Common Data Binding: Add the Update Data article (#260)
DevExpress/devextreme-documentation: concepts/70 Data Binding/03 Update Data/05 DevExtreme DataSource/10 Data Modification.md

> Data modification is implemented by the store and its methods. To call them, you need a store instance that you can get using the **DataSource**'s [store()](/api-reference/30%20Data%20Layer/DataSource/3%20Methods/store().md '/Documentation/ApiReference/Data_Layer/DataSource/Methods/#store') method.

Reviewer: ```suggestion
Data modification is implemented by the store and its methods. To call these methods, use the **DataSource**'s [store()](/api-reference/30%20Data%20Layer/DataSource/3%20Methods/store().md '/Documentation/ApiReference/Data_Layer/DataSource/Methods/#store') method to get a store instance.
```

            store.remove(1)
                .then(
                    (key) => { /* ... */ },
                    (error) => { /* ... */ }
                );

            dataSource.load();
        }
    }
    </script>

##### React

    <!-- tab: App.js -->
    import React from 'react';
    import DataSource from 'devextreme/data/data_source';

    const dataSource = new DataSource({
        // ...
    });

    class App extends React.Component {
        constructor(props) {
            super(props);

            const store = dataSource.store();
            store.insert({ id: 1, name: "John Doe" })
                .then(
                    (dataObj) => { /* ... */ },
                    (error) => { /* ... */ }
                );

            store.update(1, { name: "John Smith" })
                .then(
                    (dataObj) => { /* ... */ },
                    (error) => { /* ... */ }
                );

            store.remove(1)
                .then(
                    (key) => { /* ... */ },
                    (error) => { /* ... */ }
                );

            dataSource.load();
        }

        // ...
    }
    export default App;

---

> If a component is [integrated with a push service](/Documentation/Guide/Data_Binding/Data_Layer/#Data_Modification/Integration_with_Push_Services), use the [push(changes)](/api-reference/30%20Data%20Layer/Store/3%20Methods/push(changes).md '/Documentation/ApiReference/Data_Layer/ArrayStore/Methods/#pushchanges') method for all operations instead. You can also use it to edit local data. This method accepts an array, so you can make multiple changes in one method call.

Reviewer 2: It cannot be `instead`. You can use push to make notifications in widgets but not for data updates if remote data is used.

Reviewer 2: Also, for me, it's still better to keep Push API separate like we have now
https://js.devexpress.com/Documentation/Guide/Data_Layer/Data_Layer/#Data_Modification/Integration_with_Push_Services
Or it should be referenced as more detailed explanation in the second part of this topic
